apps/src/studio/levels.js

```diff
@@ -2010,6 +2040,12 @@ levels.js_hoc2015_event_four_items = {
     'imageWidth': 100,
     'imageHeight': 100
   },
+
+  'progressConditions' : [
+    { required: { 'timedOut': true }, 
+      result: { 'success': false, message: msg.failedFourItemsTimeout() } }
+  ],
```

Comment:
In these earlier levels, should there be a success case, or should we never check until the program stops running?

Also, should treating the hazard as a failure condition be tied into this system now?